query-parser/src/parser_util.rs

```diff
@@ -40,6 +40,7 @@ macro_rules! matches_plain_symbol {
 /// The provided `$body` will be evaluated with `$input` bound to the input stream.
 ///
 /// `$body`, when run, should return a `ParseResult` of the appropriate result type.
+#[macro_export]
```

Comment:
The macros are built on each other, so you can't move and use `d_v_s_p_f` without moving the others.